src/message/html/renderMessagesAsHtml.js

```diff
@@ -8,15 +8,16 @@ import timeRowAsHtml from './timeRowAsHtml';
 export default ({ auth, subscriptions, renderedMessages, isFetching, narrow, doNarrow }) =>
   renderedMessages.reduce(
     (list, section, index) => {
-      list.push(
-        messageHeaderAsHtml({
-          auth,
-          item: section.message,
-          subscriptions,
-          narrow,
-        }),
-      );
-
+      if (Object.keys(section.message).length > 0) {
```

Comment:
Doing this on html side will be more tricky & become complex. Should I do in that way?

Reply:
Not sure what you mean by 'html side'.
In any case, handling this in `messageHeaderAsHtml` is the better option (it will check for length === 0 as first line and return '' if true)